### DISCUSSION TITLE
👋 Take the .NET CLI survey to help shape the future of .NET tooling!

### DISCUSSION THREAD

**Discussion by JonDouglas** [dotnet/core#9387] [General]
Hey friends,

We're excited to announce the [launch of a survey](https://www.surveymonkey.com/r/MDX9B5G?sessionId=[sessionId_value]) aimed at gathering your insights and feedback on the .NET CLI. As passionate .NET developers, your experiences and suggestions are crucial in helping us improve and enhance the tools you use every day.

Your input will directly influence the development and direction of the .NET CLI, ensuring it meets the growing needs of the developer community. This is your chance to voice your opinions, share your challenges, and suggest features that could make your development experience even better.

Take a few minutes to share your thoughts:

[![image](https://github.com/dotnet/core/assets/1288848/6a5a7ce2-6bce-4cf4-849c-5023667a2f87)](https://www.surveymonkey.com/r/MDX9B5G?sessionId=[sessionId_value])

**Comment by KennethHoff** [comment]
You tricked me with that first page contact info, thinking it was the last page 😂

Good survey though 👍🏻

**Comment by baronfel** [comment]
Thanks folks, we're closing the survey now. Keep an eye out for a blogpost about the findings soon!
